benchmark-ph1ex1/2D-model/2D-fullcore26G.i

```diff
@@ -0,0 +1,172 @@
+[GlobalParams]
+  num_groups = 26
+  num_precursor_groups = 8
+  use_exp_form = false
+  group_fluxes = 'group1 group2 group3 group4 group5 group6 group7 group8 group9 group10 group11 group12 group13 group14 group15 group16 group17 group18 group19 group20 group21 group22 group23 group24 group25 group26'
+  sss2_input = true
+  account_delayed = false
+  temperature = 750
+[]
+
+[Mesh]
+  file = '2D-fullcore.msh'
+[../]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[Nt]
+  var_name_base = group
+  vacuum_boundaries = 'ref_bot ref_top wall3'
+  create_temperature_var = false
+  dg_for_temperature = false
+  eigen = true
+[]
+
+[Materials]
+  [./fuel]
+    type = GenericMoltresMaterial
+    property_tables_root = '../oecdxsA-26G/mhtgr_fuel_'
+    interp_type = 'linear'
+    block = 'fuel'
+  [../]
+  [./ireflector]
+    type = GenericMoltresMaterial
+    property_tables_root = '../oecdxsA-26G/mhtgr_ireflector_'
+    interp_type = 'linear'
+    block = 'ireflector'
+  [../]
+  [./oreflector]
+    type = GenericMoltresMaterial
+    property_tables_root = '../oecdxsA-26G/mhtgr_oreflector_'
+    interp_type = 'linear'
+    block = 'oreflector'
+  [../]
+  [./breflector]
+    type = GenericMoltresMaterial
+    property_tables_root = '../oecdxsA-26G/mhtgr_breflector_'
+    interp_type = 'linear'
+    block = 'breflector'
+  [../]
+  [./treflector]
+    type = GenericMoltresMaterial
+    property_tables_root = '../oecdxsA-26G/mhtgr_treflector_'
+    interp_type = 'linear'
+    block = 'treflector'
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = InversePowerMethod
+
+  # normalization = total_fission_heat
```

Comment:
Do not commit commented lines. They can be retrieved from the git history. Leaving these comments clutters the file and can easily mislead the user about what was simulated.
```suggestion
```